pkg/lsp/service.go

```diff
@@ -200,9 +201,36 @@ type Hover struct {
 	Range    *Range         `json:"range,omitempty"`
 }
 
-type MarkedString struct {
+type MarkedString markedString
+
+type markedString struct {
```

Comment:
I wanted to take advantage of the default behavior of `json.Unmarshal` (less code to update if we later add more fields to the struct). This is the best I could come up with, but happy to amend if there's a better way.